django/thunderstore/usermedia/tests/test_cleanup.py

```diff
@@ -69,3 +76,17 @@ def test_usermedia_cleanup() -> None:
             )
             is not None
         )
+    with pytest.raises(
```

Comment:
This is a somewhat pointless assertion addition and could probably be removed. Instead, it should be asserted that `bad_upload_id_upload` no longer exists in the database after `cleanup_expired_uploads()` has been called.

Reply:
I kept the assert, but added the object db check 👍